pyperf/_runner.py

```diff
@@ -305,10 +305,13 @@ def _process_args_impl(self):
             self._only_in_worker("--worker-task")
 
         if args.tracemalloc:
-            try:
-                import tracemalloc   # noqa
-            except ImportError as exc:
-                raise CLIError("fail to import tracemalloc: %s" % exc)
+            if args.action == 'command':
+                raise CLIError('--tracemalloc cannot be used with pyperf command')
+            else:
+                try:
+                    import tracemalloc   # noqa
+                except ImportError as exc:
+                    raise CLIError("fail to import tracemalloc: %s" % exc)
```

Comment:
Please remove the else and just leave the code as it is. Just add your if+raise.